.github/workflows/ci.yml

```diff
@@ -175,24 +179,26 @@ jobs:
         run: pnpm run test:e2e
 
   smoke:
-    name: 'Test (Smoke): ${{ matrix.os }} (node@${{ matrix.node_version }})'
+    name: 'Test (Smoke): ${{ matrix.os }} (node@${{ matrix.NODE_VERSION }})'
     runs-on: ${{ matrix.os }}
     needs: build
     strategy:
       matrix:
-        os: [ubuntu-latest]
-        node_version: [14]
+        OS: [ubuntu-latest]
+        NODE_VERSION: [14]
+    env:
+      NODE_VERSION: ${{ matrix.NODE_VERSION }}
```

Comment:
Expose current NODE_VERSION as env variable, used by Turborepo to calculate hash